examples/example-gateway/build/clients/baz/baz.go

```diff
@@ -0,0 +1,166 @@
+// Code generated by zanzibar
+// @generated
+
+// Package bazClient is generated code used to make or handle TChannel calls using Thrift.
+package bazClient
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/uber/zanzibar/runtime"
+
+	clientsBazBaz "github.com/uber/zanzibar/examples/example-gateway/build/gen-code/clients/baz/baz"
+)
+
+// Interfaces for the service and client for the services defined in the IDL.
+
+// TChanBaz is the interface that defines the server handler and client interface.
+type TChanBaz interface {
+	Call(
+		ctx context.Context,
+		reqHeaders map[string]string,
+		args *clientsBazBaz.SimpleService_Call_Args,
+	) (
```

Comment:
fix in https://github.com/uber/zanzibar/pull/57/commits/7d75a0d649c32002fbdba3b6acf2b53b74fb8dfa